backend/src/nodes/properties/outputs/numpy_outputs.py

```diff
@@ -96,10 +96,18 @@ def get_broadcast_data(self, value: np.ndarray):
         img = value
         h, w, c = get_h_w_c(img)
 
-        base64_img = preview_encode(img, 512)
+        # Encode for multiple scales. Use the preceding scale to save time encoding the smaller sizes.
+        base64_2048, np_2048 = preview_encode(img, 2048)
+        base64_1024, np_1024 = preview_encode(np_2048, 1024)
```

Comment:
Using the downscaled image from the previous iteration is a really clever trick. I like that a lot.